projects/canopy/src/lib/forms/radio/style-variants/filter-variant.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../../styles/mixins';
+
+.lg-radio-button--filter {
```

Comment:
should we modify the existing radio button component css to have a similar top level rule?
```
.lg-radio-button--radio {
```
I'm just wondering if we want to completely isolate the two variants from a css point of view, or if we are happy that the default radio one could impact this one?

Reply:
Yea, I would also go with complete isolation.

Reply:
Done.